Policies/Network/allow-azurefirewallpolicy-user-defined-standard-ports-fqdns-within-application-rules/azurepolicy.json

```diff
@@ -0,0 +1,131 @@
+{
+    "displayName": "Azure firewall policy should only allow user defined standard ports and FQDNs within application rules",
+    "description": "Enforce usage of user defined standard ports and FQDNs (default deny wildcard '*'). This is a common requirement in many regulatory and industry compliance standards.", 
+    "mode": "All",
+    "policyRule": {
+      "if": {
+        "allOf": [
+          {
+            "field": "type",
+            "equals": "Microsoft.Network/firewallPolicies/ruleCollectionGroups"
+          },
+          {
+            "count": {
+              "field": "Microsoft.Network/firewallPolicies/ruleCollectionGroups/ruleCollections[*]",
+              "where": {
+                "anyOf": [
+                  {
+                    "field": "Microsoft.Network/firewallPolicies/ruleCollectionGroups/ruleCollections[*].priority",
+                    "lessOrEquals": "[parameters('priority')]"
+                  },
+                  {
+                    "count": {
+                      "field": "Microsoft.Network/firewallPolicies/ruleCollectionGroups/ruleCollections[*].FirewallPolicyFilterRuleCollection.rules[*]",
+                      "where": {
+                        "allOf": [
+                          {
+                            "count": {
+                              "field": "Microsoft.Network/firewallPolicies/ruleCollectionGroups/ruleCollections[*].FirewallPolicyFilterRuleCollection.rules[*].ApplicationRule.targetFqdns[*]",
+                              "where": {
+                                "field": "Microsoft.Network/firewallPolicies/ruleCollectionGroups/ruleCollections[*].FirewallPolicyFilterRuleCollection.rules[*].ApplicationRule.targetFqdns[*]",
+                                "contains": "*"
+                              }
```

Comment:
same comment as the one regarding the priority- consider having this as a separate policy